src/ScottPlot/Palette.cs

```diff
@@ -27,7 +27,8 @@ public static class Palette
         public static ScottPlot.Drawing.Palette OneHalfDark => new(new ScottPlot.Drawing.Colorsets.OneHalfDark());
         public static ScottPlot.Drawing.Palette PolarNight => new(new ScottPlot.Drawing.Colorsets.PolarNight());
         public static ScottPlot.Drawing.Palette SnowStorm => new(new ScottPlot.Drawing.Colorsets.Snowstorm());
-        public static ScottPlot.Drawing.Palette xgfs25 => new(new ScottPlot.Drawing.Colorsets.Tsitsulin());
+        public static ScottPlot.Drawing.Palette Redness => new(new ScottPlot.Drawing.Colorsets.Redness());
```

Comment:
If this is supposed to be in alphabetical order then Redness and SnowStorm should be swapped.

Reply:
> If this is supposed to be in alphabetical order then Redness and SnowStorm should be swapped.

Makes sense, I will change that in a bit.

Reply:
I reordered the method call but then I needed to get the changes from master, hopefully, will not break anything, let me know, thanks.

Reply:
Looks good to me